config.js

```diff
@@ -18,10 +17,15 @@ try {
 try {
   if (!config) {
     config = fs.readFileSync(path.resolve(relativeRoot, '.mock-yeah'));
-    config = JSON.parse(config);
   }
 } catch (err) {
   // noop
 }
 
+try {
+  if (config) config = JSON.parse(config);
+} catch (err) {
+  throw new Error('.mockyeah configuration file JSON invalid.');
```

Comment:
Would `Invalide JSON in .mockyeah configuration file`, read better? 

Otherwise, this is great!